apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
I tried modifying Clouds Maximum Percent and really saw no difference. On Windows at least, the moon texture didn't appear to be impacted by setting the value to 0. Values in Morrowind.ini vary from 1.0, to 0.66.